.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "\U0001F914 Question"
+about: Usage question that isn't answered in docs or discussion
+
+---
+
+## Question
+
+Before asking a question, make sure you have:
+
+- Searched existing Stack Overflow questions.
+- Googled your question.
+- Searched open and closed [GitHub issues](https://github.com/pingcap/tidb-operator/issues?utf8=%E2%9C%93&q=is%3Aissue)
+- Read the documentation:
+  - [TiDB Operator Readme](https://github.com/pingcap/tidb-operator)
+  - [TiDB Operator Doc](https://github.com/pingcap/tidb-operator/tree/master/docs)
```

Comment:
This is being moved soon?

Reply:
Yes, we'll move our documents to https://github.com/pingcap/docs and https://github.com/pingcap/docs-cn to make our documents available on PingCAP official website.

Reply:
Then this link will break?

Reply:
We should update the links when our documents are all moved to the pingcap/docs and pingcap/docs-cn.